check_advisories.py

```diff
@@ -28,6 +28,7 @@
 GIT = os.getenv('GIT_BIN', 'git')
 ADVISORIES_DIR = 'announce'
 HOF_DIR = 'bug-bounty-hof'
+CVE_RE = re.compile('^CVE-20[0-9]{2}-[0-9]+$')
```

Comment:
According to the CVE site it seems that the sequence section at the end is at least 4 digits, and up to 7 but could be more. Should we be more specific here and do `[0-9]{4,9}`?

Reply:
Maybe it's not a big deal and we are only interested in avoiding the place-holder `XXXX`?

Reply:
Updated!